apps/web/components/connected/Landspitali/MemorialCard.tsx

```diff
@@ -0,0 +1,363 @@
+          <Text>
+            {formatMessage(m.overview.recipientName)} {data.recipientName}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.recipientAddress)} {data.recipientAddress}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.recipientPostalCode)}{' '}
+            {data.recipientPostalCode}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.inMemoryOf)} {data.inMemoryOf}
+          </Text>
+        </Stack>
+        <Stack space={2}>
+          <Text variant="h3">
+            {formatMessage(m.overview.senderSignatureTitle)}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.senderSignature)} {data.senderSignature}
+          </Text>
+        </Stack>
+        <Stack space={2}>
+          <Text variant="h3">
+            {formatMessage(m.overview.memorialCardTitle)}
+          </Text>
+          <Text>
+            {formatMessage(m.overview.amountISK)} {data.amountISK} kr.
+          </Text>
+          <Text>
+            {formatMessage(m.overview.fund)} {data.fund}
+          </Text>
+        </Stack>
+        <Box display="flex" justifyContent="spaceBetween">
+          <Button onClick={handleReset}>
+            {formatMessage(m.overview.goBack)}
+          </Button>
+          <Button>{formatMessage(m.overview.pay)}</Button>
+        </Box>
+      </Stack>
+    )
+  }
+
+  const requiredRule = {
+    required: { value: true, message: 'Það þarf að fylla út þennan reit' },
+  }
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Stack space={4}>
+          <SelectController
+            id="fund"
+            name="fund"
+            label={formatMessage(m.info.fundLabel)}
+            options={fundOptions}
+            defaultValue={fundOptions.find(
+              (opt: { value: string }) => opt.value === watch('fund'),
+            )}
+            onSelect={(opt) => setValue('fund', (opt?.value as string) || '')}
+            error={errors.fund?.message}
+            rules={requiredRule}
+          />
+
+          <InputController
+            id="inMemoryOf"
+            control={control}
+            label={formatMessage(m.info.inMemoryOfLabel)}
+            size="xs"
+            error={errors.inMemoryOf?.message}
+            rules={requiredRule}
+          />
+
+          <Text variant="h2">{formatMessage(m.info.amountOfMoneyTitle)}</Text>
+          <Text>{formatMessage(m.info.amountOfMoneyExtraInfo)}</Text>
+
+          <Box display="flex" flexWrap="wrap" columnGap={2} rowGap={2}>
+            {PRESET_AMOUNTS.map((amount) => (
+              <FocusableBox
+                key={amount}
+                border="standard"
+                borderRadius="standard"
+                padding={2}
+              >
+                <RadioButton
+                  id={`amount-${amount}`}
+                  name="amountISK"
+                  label={parseInt(amount)}
```

Comment:
Let's show a dot as a thousands separator like it is today at https://www.landspitali.is/um-landspitala/stydjum-starfsemina/minningarkort/

Reply:
5.000
10.000

and so on